pom.xml

```diff
@@ -23,9 +23,14 @@
 			<artifactId>spring-boot-starter-web</artifactId>
 		</dependency>
 <!--		uncomment while you add database details-->
+		<dependency>
+			<groupId>org.springframework.boot</groupId>
+			<artifactId>spring-boot-starter-data-jpa</artifactId>
+		</dependency>
 <!--		<dependency>-->
-<!--			<groupId>org.springframework.boot</groupId>-->
-<!--			<artifactId>spring-boot-starter-data-jpa</artifactId>-->
+<!--			<groupId>javax.validation</groupId>-->
+<!--			<artifactId>validation-api</artifactId>-->
+<!--			<version>2.0.1.Final</version> &lt;!&ndash; Use the appropriate version &ndash;&gt;-->
```

Comment:
You can rem9ve this dependency

Reply:
I put that in comments. Will remove this.